aws/ec2.go

```diff
@@ -92,6 +93,11 @@ func nukeAllEc2Instances(session *session.Session, instanceIds []*string) error
 		return errors.WithStackTrace(err)
 	}
 
+	if !waitForTermination {
```

Comment:
Had added it to support async deletion. Reverted it back as it might be better tackled in a separate issue. Sometimes, we just want to trigger deletion and not wait for resources that take a long time to nuke.